src/controllers/mintController.test.ts

```
@@ -125,6 +125,28 @@ describe("mintController", () => {
     expect(err.message).toBe("Wallet address does not match user");
   });
 
+  it("rejects unsupported basket deposit currencies with a clear error", async () => {
```

ESLint

[error] Delete `␍`

(prettier/prettier)